panel/widgets/chat.py

```diff
@@ -880,14 +884,14 @@ def _stream(self, token: str, entry: ChatEntry) -> ChatEntry:
             if hasattr(value, "objects"):
                 parent_panel = value
                 attr = "objects"
-                value = value.objects[i]
+                value = getattr(value, "objects")[i]
                 i = -1
             elif hasattr(value, "object"):
                 attr = "object"
-                value = value.object
+                value = getattr(value, "object")
             elif hasattr(value, "value"):
                 attr = "value"
-                value = value.value
+                value = getattr(value, "value")
```

Comment:
This makes the code more obscure and has the same effect as the previous code.

Reply:
I sort of agree. The alternative would be to silende static Analysis on that line. I prefer to avoid silencing when possible. But its a Trade off.

Reply:
The functionality code-wise is identical. What is it complaining about?

Reply:
Much prefer silencing here, changing runtime code because of a typing issue is an anti-pattern imo.

Reply:
Well that's not quite true, changing runtime code because of a typing issue often indicates some actual issue with the code but in this case it's an issue with Python typing being dumb.

Reply:
Yes. But in order to be able to get value from static analysis, you need to remove `False` positives. Otherwise you and the next developer will be investigating the issue over and over again.

And you can argue that disabling static analysis in a line creates a sort of debt because you never know whether its silenced as a "quick fix" by someone or whether the silencing is correct.

That is why I have a tendency to find a way of writing the code such that I don't need to silence the static analysis.